pkg/dpll/dpll.go

```diff
@@ -334,14 +335,23 @@ func (d *DpllConfig) nlUpdateState(replies []*nl.DoDeviceGetReply) bool {
 			glog.Infof("discarding on clock ID %v (%s): ", nl.GetDpllStatusHR(reply), d.iface)
 		}
 	}
+	for _, pin := range pins {
+		// TODO - figure out pin ID per clock ID in the device plugin?
+		// Meanwhile there are only the relevant pins that report phase offset
```

Comment:
We need to find a way to know which pin IDs are relevant for every clock ID.
I don't see an elegant way to do it, other than provide it in a configuration. This will make it harder and more error-prone to configure the daemon, but will make the the intention explicit. 
Any other thoughts?

Reply:
I will try to look into the ICE code to see how they attain pin IDs.